web/ui/dashboard/src/app/private/components/create-project-component/create-project-component.component.ts

```diff
@@ -274,4 +275,10 @@ export class CreateProjectComponent implements OnInit {
 		this.confirmModal = true;
 		document.getElementById('projectForm')?.scroll({ top: 0, behavior: 'smooth' });
 	}
+
+	confirmAction() {
+		this.projectForm.get('config.disable_endpoint')?.valueChanges.subscribe(selectedValue => {
```

Comment:
Do you need to subscribe to get this value? @Oluwadaminiola

Reply:
yes

Reply:
it's an easy way to detect if any change has been made to that formcontrol value

Reply:
Then you should unsubscribe after @Oluwadaminiola

Reply:
okay, I'll do that